### PR TITLE
[UI] Show pull icon on pull

### PR DESCRIPTION
**Pull request by 6543**
## Before
Bildschirmfoto zu 2020-01-29 18-49-28

## After
Bildschirmfoto zu 2020-01-29 18-49-08

### CODE REVIEW

File: modules/graceful/manager_windows.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ type Manager struct {
 	runningServerWaitGroup sync.WaitGroup
 	createServerWaitGroup  sync.WaitGroup
 	terminateWaitGroup     sync.WaitGroup
-	shutdownRequested	   chan struct{}
+	shutdownRequested      chan struct{}
```
Comment (guillep2k) [inline modules/graceful/manager_windows.go:46]: Please remove unrelated change.

Comment (6543) [reply modules/graceful/manager_windows.go:46]: @guillep2k should I send a own PR for this one-line change?!?

**Review by guillep2k — APPROVED** [review]
Please ignore my comment about the unrelated change. `make fmt` must have done that (as for why it was accepted without it before, it beats me).

File: modules/graceful/manager_windows.go (hunk 1 above)
Comment (6543) [reply modules/graceful/manager_windows.go:46]: It's just a formation issue and since **this** PR is smal I think it fits in well

Comment (guillep2k) [reply modules/graceful/manager_windows.go:46]: > 
> 
> @guillep2k should I send a own PR for this one-line change?!?

@6543 I guess this change comes from `make fmt` itself so it should be OK. But if it was not, what I usually do is to skip unrelated changes from `git add`.

**Review by jolheiser — APPROVED** [review]

**Comment by 6543** [comment]
should I send a backport (only templates/repo/issue/view_title.tmpl change) to 1.11.x branch?

**Comment by zeripath** [comment]
was it previously a pull icon and then it changed or is this a new feature?

**Comment by zeripath** [comment]
if it's a new feature probably we shouldn't backport

**Comment by 6543** [comment]
Ibwould considder this as ui-fix in my opinion

**Comment by zeripath** [comment]
send a backport then
